To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
+++ ../src-work/minecraft/net/minecraft/client/gui/recipebook/GuiRecipeOverlay.java
@@ -230,11 +230,11 @@
             int k = 3;
To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
 
-            if (this.field_193924_p instanceof ShapedRecipes)
+            if (this.field_193924_p instanceof net.minecraftforge.common.crafting.IShapedRecipe)
To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
-                ShapedRecipes shapedrecipes = (ShapedRecipes)this.field_193924_p;
-                k = shapedrecipes.func_192403_f();
-                l = shapedrecipes.func_192404_g();
To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
+                k = shapedrecipes.getRecipeWidth();
+                l = shapedrecipes.getRecipeHeight();
             }
To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
             Iterator<Ingredient> iterator = this.field_193924_p.func_192400_c().iterator();
